cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -122,12 +122,43 @@
     If you come across a script that makes a convincing case to be included in
     the head, then file an issue or PR to discuss including it.
 -->
-    {# Google Tag Manager #}
+
+    <!-- Google Optimize page-hiding snippet -->
+    <style>
```

Comment:
I tried to explain it succintly in the PR description, but I can give some more details if you like.

Three weeks ago, we removed the explicit call to Google Tag Manager from the head of You Have the Right: https://[GHE]/CFGOV/you-have-the-right/pull/27

At that time, the number of Google Optimize sessions for the alternate headline test we were running on that page dropped significantly, from roughly 90-95% of the total traffic to that page (as reported by Analytics), to roughly 25% of it.

In trying to find out why, we discovered that our implementation for using all three of these services together (a single Google Tag Manager snippet in our template) is not the recommended implementation by Google (an Analytics snippet that also calls in Optimize, with its standard Analytics pageview event removed, _plus_ the standard GTM snippet).

Adding the page-hiding snippet is unrelated to that particular issue, but it was an additional thing they recommend doing to prevent a flash between rendering the original page content and rendering the test variation.

Does that help clear it up?